front/src/App.js

```diff
@@ -12,13 +12,19 @@ import EliminationsBrackets from "./components/EliminationsBrackets/Eliminations
 import AdminCompetitionsMenu from "./components/AdminPanel/CompetitionsMenu";
 import AdminNewCompetition from "../src/components/AdminPanel/NewCompetition/NewCompetition";
 import AdminPanel from "./components/AdminPanel/Panel/Panel";
+import EditCompetitionDetails from "./components/AdminPanel/Panel/CompetitionDetails/EditDetails/EditDetails"
 
 function App() {
     return (
         <div>
             <Layout>
                 <Switch>
-                    <Route path="/admin/panel/:id" component={AdminPanel} />
+                    <Route 
+                        path="/admin/panel/:id/details/edit" 
+                        component={EditCompetitionDetails} />
```

Comment:
I think that this route should be moved to Panel.js, where other admin/panel/:id/... routes are defined